examples/simple-kafkacluster.yaml

```diff
@@ -8,14 +8,20 @@ spec:
   zookeeperReference:
     namespace: default
     name: simple
-  opaReference:
-    namespace: default
-    name: simple-opacluster
+  opa:
+    reference:
+      namespace: default
+      name: simple-opacluster
+    authorizerClassName: com.bisnode.kafka.authorization.OpaAuthorizer
```

Comment:
I think we can leave this as is for now, but this setting should not be exposed to the user by default, instead this should be derived if a opa reference is specified.
That being said I think we can take care of that in a subsequent PR.

Reply:
Ok agree.